SPECS/prebuilt-ca-certificates/prebuilt-ca-certificates.spec

```diff
@@ -0,0 +1,45 @@
+Summary:        Prebuilt version of ca-certificates package.
+Name:           prebuilt-ca-certificates
+Version:        20200720
```

Comment:
Please also add a comment inside `ca-certificates.spec` saying, that the version from `prebuilt-ca-certificates` should always be kept aligned with the one from `ca-certificates`. It seems like a better place to put it, since I wouldn't expect the version to change on its own here.